deployment/GP05MOAS-GL363_Deploy.csv

```diff
@@ -1,9 +1,13 @@
-RB-16-05,,RB-16-05,,GP05MOAS-GL363-00-ENG000000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,50.16648,-144.50107,,1000,1000,
-RB-16-05,,RB-16-05,,GP05MOAS-GL363-01-FLORDM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,50.16648,-144.50107,,1000,1000,
-RB-16-05,,RB-16-05,,GP05MOAS-GL363-02-DOSTAM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00161,50.16648,-144.50107,,1000,1000,
-RB-16-05,,RB-16-05,,GP05MOAS-GL363-04-CTDGVM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,50.16648,-144.50107,,1000,1000,
\ No newline at end of file
+MV1309,,MV1404,,GP05MOAS-GL363-00-ENG000000,1,1,2013-07-19T23:00:00,2014-06-15T17:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,50.10807,-144.46093,,1000,1000,
+MV1309,,MV1404,,GP05MOAS-GL363-01-FLORDM000,1,1,2013-07-19T23:00:00,2014-06-15T17:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,50.10807,-144.46093,,1000,1000,
+MV1309,,MV1404,,GP05MOAS-GL363-02-DOSTAM000,1,1,2013-07-19T23:00:00,2014-06-15T17:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00161,50.10807,-144.46093,,1000,1000,
+MV1309,,MV1404,,GP05MOAS-GL363-04-CTDGVM000,1,1,2013-07-19T23:00:00,2014-06-15T17:19:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,50.10807,-144.46093,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-00-ENG000000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-01-FLORDM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-02-DOSTAM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00161,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-04-CTDGVM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,50.16648,-144.50107,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-00-ENG000000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-01-FLORDM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-02-DOSTAM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-04-CTDGVM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01577,-133.00577,,1000,1000,

```

Comment:
The recovery on 10/2/2020 at 19:50 is not indicated in the QLR.  Which one is right? Was it recovered or not?

Reply:
Updated QLR to include recovery time for 363.

Reply:
Looks good... sadly...